<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>baseRotateDialog</class>
 <widget class="QDialog" name="baseRotateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>92</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>旋转翻转</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="rotateClockWiseBtn">
     <property name="text">
      <string>顺时针旋转</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="rotateVerFlipBtn">
     <property name="text">
      <string>垂直翻转</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="rotateAntiClockWiseBtn">
     <property name="text">
      <string>逆时针旋转</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="rotateHorFlipBtn">
     <property name="text">
      <string>水平翻转</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
